cts/CTStests.py

```diff
@@ -918,7 +920,11 @@ def __call__(self, node):
 
     def errorstoignore(self):
         '''Return list of errors which should be ignored'''
-        return [ """cib:.*readCibXmlFile:""", """HA_VALGRIND_ENABLED""" ]
+        return [
+            r"cib.*: \*\*\*\*\*\*\*\*\*\*\*\*\*",
```

Comment:
So previously it was ignoring "cib:.*readCibXmlFile:" (i.e. all errors from that function). Without the function name, that becomes tricky. I looked at the possible errors from the function, and my feeling is that the previous ignore was overbroad, and we really just want to ignore this:

            crm_err("*********************************************************");
            crm_err("*** Disabling disk writes to avoid confusing Valgrind ***");
            crm_err("*********************************************************");

I'm not a fan of an error message of entirely stars :-) but I didn't want to change the messages themselves in this revision if I could avoid it.